src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -15,7 +15,8 @@ const en = ({ isDivorce, isApplicant2 }: CommonContent) => ({
     isDivorce ? 'divorce application' : 'application to end your civil partnership'
   } (PDF)</a>`,
   reviewContactDetails: `<a class="govuk-link" href="${
-    (isApplicant2 ? APPLICANT_2 : '') + REVIEW_CONTACT_DETAILS
+    (isApplicant2 ? (userCase?.applicationType === ApplicationType.SOLE_APPLICATION ? RESPONDENT : APPLICANT_2) : '') +
```

Comment:
is there a valid scenario where applicationType can be null, if not we can remove '?'

Reply:
working on the ? in another branch